app_feup/lib/controller/load_info.dart

```diff
@@ -14,13 +14,16 @@ import 'package:uni/redux/refresh_items_action.dart';
 import 'local_storage/app_shared_preferences.dart';
 
 Future loadReloginInfo(Store<AppState> store) async {
-  final Tuple2<String, String> userPersistentInfo =
-      await AppSharedPreferences.getPersistentUserInfo();
+  final Tuple3<String, String, List<String>> userPersistentInfo =
+      await AppSharedPreferences.getPersistentUserInfoFac();
   final String userName = userPersistentInfo.item1;
   final String password = userPersistentInfo.item2;
-  if (userName != '' && password != '') {
+  final List<String> faculties = userPersistentInfo.item3;
+  if (userName != '' && password != '' && faculties != []) {
```

Comment:
We need to have into account the migration of the current users. The `faculties` object may be empty. If that's the case, we must store in the shared preferences the faculty `feup`.

Reply:
Solved